### PR TITLE
100 changes to embed view locking during card payment

### PR DESCRIPTION
**Pull request by mntzrr**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -17,47 +17,47 @@
   dependencies:
     "@babel/highlight" "^7.18.6"
 
-"@babel/compat-data@^7.17.7", "@babel/compat-data@^7.18.8":
-  version "7.18.8"
-  resolved "https://registry.yarnpkg.com/@babel/compat-data/-/compat-data-7.18.8.tgz#2483f565faca607b8535590e84e7de323f27764d"
-  integrity sha512-HSmX4WZPPK3FUxYp7g2T6EyO8j96HlZJlxmKPSh6KAcqwyDrfx7hKjXpAW/0FhFfTJsR0Yt4lAjLI2coMptIHQ==
+"@babel/compat-data@^7.17.7", "@babel/compat-data@^7.18.8", "@babel/compat-data@^7.19.3":
```
Comment (frodeaa) [inline yarn.lock:20]: why do we bump dev dependencies in this PR?

File: assets/js/dintero-checkout-express.js [hunk 2]
```diff
@@ -386,18 +385,55 @@ jQuery( function( $ ) {
...
-		 * Submit the order using the WooCommerce AJAX function.
-		 *
-		 * @param {callback} callback
+		 * Block form fields from being modified by the user.
 		 */
-		submitOrder( callback ) {
+		blockForm() {
+			/* Order review. */
 			$( '.woocommerce-checkout-review-order-table' ).block( {
+				message: dinteroCheckoutParams.pip_text,
```
Comment (frodeaa) [inline assets/js/dintero-checkout-express.js:393]: can you include a screenshot of how the form will look when blocked with this change?

**Review by frodeaa — APPROVED** [review]

File: assets/js/dintero-checkout-express.js (hunk 2 above)
Comment (frodeaa) [reply assets/js/dintero-checkout-express.js:393]: @svnv Any feedback to this?

File: assets/css/dintero-checkout-express.css [hunk 3]
```diff
@@ -1,3 +1,17 @@
+.dintero-checkout-pip {
+    display: block;
+    font-size: 0.9em;
+    box-shadow: 1px 1px 5px -1px grey;
```
Comment (svnv) [inline assets/css/dintero-checkout-express.css:4]:
```suggestion
    box-shadow: 1px 1px 5px -1px grey;
    padding: 5px;
    border-radius: 5px;
```

File: assets/js/dintero-checkout-express.js (hunk 2 above)
Comment (svnv) [reply assets/js/dintero-checkout-express.js:393]: Added a suggestion with some added padding and border radius the same as in the checkout.

**Pull request merged**